frontend/src/components/ScrollerComponents.js

```diff
@@ -47,7 +48,7 @@ class ScrollButton extends Component {
         // colors: ["#044747", "#079191", "#38adad", "#90e3e3", "#d5f7f7"]
       }
     }
-  
+
     render () {
       return <div className="long">
       {
```

Comment:
I feel like the scroll speed should be faster. Setting scrollStepInPx to around 150 seems to be a good speed.

Reply:
Yes sure I can adjust that, also I am looking into making it fade in/fade out depending on how far up/down the user is.

Reply:
Yes, that would be a great addition.